bip-0146.mediawiki

```diff
@@ -10,22 +10,40 @@
 
 ==Abstract==
 
-This document specifies proposed changes to the Bitcoin transaction validity rules to restrict signatures to using low S values.
+This document specifies proposed changes to the Bitcoin transaction validity rules to fix signature malleability.
```

Comment:
Maybe change it into "for common transaction types" ? It certainly does not entirely fix it.